UnityProjekte/Base Project/Assets/Scripts/HoverAnimation.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HoverAnimation : MonoBehaviour
+{
+    public float amplitude = 2f;
+    public float speed = 0.5f;
```

Comment:
Variable values instead of hard coded would be more flexible, but the given values should do the job